src/Swashbuckle.AspNetCore.SwaggerUI/SwaggerUIOptions.cs

```diff
@@ -63,6 +63,16 @@ public class SwaggerUIOptions
         /// Gets or sets the path or URL to the Swagger UI CSS file.
         /// </summary>
         public string StylesPath { get; set; } = "./swagger-ui.css";
+
+        /// <summary>
+        /// Expose the ConfigObject.Urls object via a new &lt;root&gt;/documentUrls
```

Comment:
I would change this to point to the property rather than having the default in the comment.

Reply:
I've updated the documentation comments to use <see> and <c> as appropriate. Let me know what you think.